app/components/Exchange/Exchange.jsx

```diff
@@ -3072,7 +3075,7 @@ class Exchange extends React.Component {
                 <div
                     className={cnames(
                         centerContainerWidth > 1200 &&
-                        (verticalOrderBook || verticalOrderBook)
+                            (verticalOrderBook || verticalOrderBook)
```

Comment:
Whole Exchange file is reformat only, please revert (rebase if necessary)

Reply:
I`ve changed https://github.com/blockchainprojects/bitshares-ui/pull/89/files/9ac96a83c547065d6e303fcbe16c8306eabb39ee#diff-3aaa4fc021c75bd17e1e59adb802e56dL2753 
Other were changed by pre-commit hook